spec/services/hashtag_autocomplete_service_spec.rb

```diff
@@ -166,6 +170,19 @@ def self.search(guardian_scoped, term, limit)
       )
     end
 
+    it "orders categories by exact match (ignoring parent/child distinction) and name then name for everything else" do
```

Comment:
I think we can clarify the exact match here, are we exact matching on slug, name or both?

Reply:
Good point, will clarify, it's just on slug for the exact match